### PR TITLE
Fix format table

### LINKED ISSUE #721: `footer` does not work with `display`
I would expect that

> For markdown-formatted tables, table footers, due to the limitation in markdown rendering, are actually just a new text line under the table.

However, it does not appear even a text line:
```r
model <- lm(mpg ~ wt + cyl, data = mtcars)
mp <- model_parameters(model)
display(mp, format = "markdown", footer = "Somethng")


|Parameter   | Coefficient |   SE |         95% CI | t(29) |      p |
|:-----------|:-----------:|:----:|:--------------:|:-----:|:------:|
|(Intercept) |       39.69 | 1.71 | (36.18, 43.19) | 23.14 | < .001 |
|wt          |       -3.19 | 0.76 | (-4.74, -1.64) | -4.22 | < .001 |
|cyl         |       -1.51 | 0.41 | (-2.36, -0.66) | -3.64 | 0.001  |
```
(Same with `format = "html"`)

### PR DESCRIPTION
**Pull request by etiennebacher**
Note: this will require a small PR on `insight` for HTML tables

``` r
library(parameters)
model <- lm(mpg ~ wt + cyl, data = mtcars)
mp <- model_parameters(model)

### HTML

display(mp, format = "html", footer = "hi", show_sigma = TRUE)
```

<div id="bmygcxnyhl" style="overflow-x:auto;overflow-y:auto;width:auto;height:auto;">
<table class="gt_table">
  <thead class="gt_header">
    <tr>
      <th colspan="6" class="gt_heading gt_title gt_font_normal gt_bottom_border" style>Model Summary</th>
    </tr>
    
  </thead>
  <thead class="gt_col_headings">
    <tr>
      <th class="gt_col_heading gt_columns_bottom_border gt_left" rowspan="1" colspan="1">Parameter</th>
      <th class="gt_col_heading gt_columns_bottom_border gt_center" rowspan="1" colspan="1">Coefficient</th>
      <th class="gt_col_heading gt_columns_bottom_border gt_center" rowspan="1" colspan="1">SE</th>
      <th class="gt_col_heading gt_columns_bottom_border gt_center" rowspan="1" colspan="1">95% CI</th>
      <th class="gt_col_heading gt_columns_bottom_border gt_center" rowspan="1" colspan="1">t(29)</th>
      <th class="gt_col_heading gt_columns_bottom_border gt_center" rowspan="1" colspan="1">p</th>
    </tr>
  </thead>
  <tbody class="gt_table_body">
    <tr><td class="gt_row gt_left">(Intercept)</td>
<td class="gt_row gt_center">39.69</td>
<td class="gt_row gt_center">1.71</td>
<td class="gt_row gt_center">(36.18, 43.19)</td>
<td class="gt_row gt_center">23.14</td>
<td class="gt_row gt_center">&lt; .001</td></tr>
    <tr><td class="gt_row gt_left">wt</td>
<td class="gt_row gt_center">-3.19</td>
<td class="gt_row gt_center">0.76</td>
<td class="gt_row gt_center">(-4.74, -1.64)</td>
<td class="gt_row gt_center">-4.22</td>
<td class="gt_row gt_center">&lt; .001</td></tr>
    <tr><td class="gt_row gt_left">cyl</td>
<td class="gt_row gt_center">-1.51</td>
<td class="gt_row gt_center">0.41</td>
<td class="gt_row gt_center">(-2.36, -0.66)</td>
<td class="gt_row gt_center">-3.64</td>
<td class="gt_row gt_center">0.001 </td></tr>
  </tbody>
  <tfoot class="gt_sourcenotes">
    <tr>
      <td class="gt_sourcenote" colspan="6">hi<br/>Residual standard deviation: 2.568 (df = 29)</td>
    </tr>
  </tfoot>
  
</table>
</div>

`display(mp, format = "html", show_sigma = TRUE)`
<div id="pamuzkvpvq" style="overflow-x:auto;overflow-y:auto;width:auto;height:auto;">
<table class="gt_table">
  <thead class="gt_header">
    <tr>
      <th colspan="6" class="gt_heading gt_title gt_font_normal gt_bottom_border" style>Model Summary</th>
    </tr>
    
  </thead>
  <thead class="gt_col_headings">
    <tr>
      <th class="gt_col_heading gt_columns_bottom_border gt_left" rowspan="1" colspan="1">Parameter</th>
      <th class="gt_col_heading gt_columns_bottom_border gt_center" rowspan="1" colspan="1">Coefficient</th>
      <th class="gt_col_heading gt_columns_bottom_border gt_center" rowspan="1" colspan="1">SE</th>
      <th class="gt_col_heading gt_columns_bottom_border gt_center" rowspan="1" colspan="1">95% CI</th>
      <th class="gt_col_heading gt_columns_bottom_border gt_center" rowspan="1" colspan="1">t(29)</th>
      <th class="gt_col_heading gt_columns_bottom_border gt_center" rowspan="1" colspan="1">p</th>
    </tr>
  </thead>
  <tbody class="gt_table_body">
    <tr><td class="gt_row gt_left">(Intercept)</td>
<td class="gt_row gt_center">39.69</td>
<td class="gt_row gt_center">1.71</td>
<td class="gt_row gt_center">(36.18, 43.19)</td>
<td class="gt_row gt_center">23.14</td>
<td class="gt_row gt_center">&lt; .001</td></tr>
    <tr><td class="gt_row gt_left">wt</td>
<td class="gt_row gt_center">-3.19</td>
<td class="gt_row gt_center">0.76</td>
<td class="gt_row gt_center">(-4.74, -1.64)</td>
<td class="gt_row gt_center">-4.22</td>
<td class="gt_row gt_center">&lt; .001</td></tr>
    <tr><td class="gt_row gt_left">cyl</td>
<td class="gt_row gt_center">-1.51</td>
<td class="gt_row gt_center">0.41</td>
<td class="gt_row gt_center">(-2.36, -0.66)</td>
<td class="gt_row gt_center">-3.64</td>
<td class="gt_row gt_center">0.001 </td></tr>
  </tbody>
  <tfoot class="gt_sourcenotes">
    <tr>
      <td class="gt_sourcenote" colspan="6">Residual standard deviation: 2.568 (df = 29)</td>
    </tr>
  </tfoot>
  
</table>
</div>

`display(mp, format = "html", footer = "hi", show_sigma = FALSE)`
<div id="iktdjcqnnu" style="overflow-x:auto;overflow-y:auto;width:auto;height:auto;">
<table class="gt_table">
  <thead class="gt_header">
    <tr>
      <th colspan="6" class="gt_heading gt_title gt_font_normal gt_bottom_border" style>Model Summary</th>
    </tr>
    
  </thead>
  <thead class="gt_col_headings">
    <tr>
      <th class="gt_col_heading gt_columns_bottom_border gt_left" rowspan="1" colspan="1">Parameter</th>
      <th class="gt_col_heading gt_columns_bottom_border gt_center" rowspan="1" colspan="1">Coefficient</th>
      <th class="gt_col_heading gt_columns_bottom_border gt_center" rowspan="1" colspan="1">SE</th>
      <th class="gt_col_heading gt_columns_bottom_border gt_center" rowspan="1" colspan="1">95% CI</th>
      <th class="gt_col_heading gt_columns_bottom_border gt_center" rowspan="1" colspan="1">t(29)</th>
      <th class="gt_col_heading gt_columns_bottom_border gt_center" rowspan="1" colspan="1">p</th>
    </tr>
  </thead>
  <tbody class="gt_table_body">
    <tr><td class="gt_row gt_left">(Intercept)</td>
<td class="gt_row gt_center">39.69</td>
<td class="gt_row gt_center">1.71</td>
<td class="gt_row gt_center">(36.18, 43.19)</td>
<td class="gt_row gt_center">23.14</td>
<td class="gt_row gt_center">&lt; .001</td></tr>
    <tr><td class="gt_row gt_left">wt</td>
<td class="gt_row gt_center">-3.19</td>
<td class="gt_row gt_center">0.76</td>
<td class="gt_row gt_center">(-4.74, -1.64)</td>
<td class="gt_row gt_center">-4.22</td>
<td class="gt_row gt_center">&lt; .001</td></tr>
    <tr><td class="gt_row gt_left">cyl</td>
<td class="gt_row gt_center">-1.51</td>
<td class="gt_row gt_center">0.41</td>
<td class="gt_row gt_center">(-2.36, -0.66)</td>
<td class="gt_row gt_center">-3.64</td>
<td class="gt_row gt_center">0.001 </td></tr>
  </tbody>
  <tfoot class="gt_sourcenotes">
    <tr>
      <td class="gt_sourcenote" colspan="6">hi<br/></td>
    </tr>
  </tfoot>
  
</table>
</div>

### Markdown

`display(mp, format = "markdown", show_sigma = TRUE)`

| Parameter   | Coefficient |  SE  |     95% CI     | t(29) |    p    |
|:------------|:-----------:|:----:|:--------------:|:-----:|:-------:|
| (Intercept) |    39.69    | 1.71 | (36.18, 43.19) | 23.14 | \< .001 |
| wt          |    -3.19    | 0.76 | (-4.74, -1.64) | -4.22 | \< .001 |
| cyl         |    -1.51    | 0.41 | (-2.36, -0.66) | -3.64 |  0.001  |

Residual standard deviation: 2.568 (df = 29)

``` r
display(mp, format = "markdown", footer = "hi", show_sigma = TRUE)
```

| Parameter   | Coefficient |  SE  |     95% CI     | t(29) |    p    |
|:------------|:-----------:|:----:|:--------------:|:-----:|:-------:|
| (Intercept) |    39.69    | 1.71 | (36.18, 43.19) | 23.14 | \< .001 |
| wt          |    -3.19    | 0.76 | (-4.74, -1.64) | -4.22 | \< .001 |
| cyl         |    -1.51    | 0.41 | (-2.36, -0.66) | -3.64 |  0.001  |

hi

Residual standard deviation: 2.568 (df = 29)

``` r
display(mp, format = "markdown", footer = "hi", show_sigma = FALSE)
```

| Parameter   | Coefficient |  SE  |     95% CI     | t(29) |    p    |
|:------------|:-----------:|:----:|:--------------:|:-----:|:-------:|
| (Intercept) |    39.69    | 1.71 | (36.18, 43.19) | 23.14 | \< .001 |
| wt          |    -3.19    | 0.76 | (-4.74, -1.64) | -4.22 | \< .001 |
| cyl         |    -1.51    | 0.41 | (-2.36, -0.66) | -3.64 |  0.001  |

hi

<sup>Created on 2022-06-02 by the [reprex package](https://reprex.tidyverse.org) (v2.0.1)</sup>

Close #721

### CODE REVIEW

**Comment by DominiqueMakowski** [comment]
I'm chiming in just to ask (I've been away for some time 😁): can we remove the footer for being printed in the console?

``` r
parameters::parameters(lm(Sepal.Width ~ Species, data=iris))
#> Registered S3 method overwritten by 'parameters':
#>   method                         from      
#>   format.parameters_distribution datawizard
#> Parameter            | Coefficient |   SE |         95% CI | t(147) |      p
#> ----------------------------------------------------------------------------
#> (Intercept)          |        3.43 | 0.05 | [ 3.33,  3.52] |  71.36 | < .001
#> Species [versicolor] |       -0.66 | 0.07 | [-0.79, -0.52] |  -9.69 | < .001
#> Species [virginica]  |       -0.45 | 0.07 | [-0.59, -0.32] |  -6.68 | < .001
#> 
#> Uncertainty intervals (equal-tailed) and p values (two-tailed) computed
#>   using a Wald t-distribution approximation.
```

<sup>Created on 2022-06-02 by the [reprex package](https://reprex.tidyverse.org) (v2.0.1)</sup>

(talking about the red "uncertainty intervals ...")

**Comment by etiennebacher** [comment]
I don't know what's best to do with this message, @strengejacke (or anyone else) what do you think?

**Comment by mattansb** [comment]
For a start, can it not be red?

**Comment by etiennebacher** [comment]
Note: there's a problem when several stats/additional things are added in `display` with HTML format. Sigma is not shown below:
``` r
library(parameters)
model <- lm(mpg ~ wt + cyl, data = mtcars)
mp <- model_parameters(model)

### HTML

display(mp, format = "html", footer = "hi", show_sigma = TRUE, show_formula = TRUE)
```

<div id="aghyxnespb" style="overflow-x:auto;overflow-y:auto;width:auto;height:auto;">

<table class="gt_table">
  <thead class="gt_header">
    <tr>
      <th colspan="6" class="gt_heading gt_title gt_font_normal gt_bottom_border" style>Model Summary</th>
    </tr>
    
  </thead>
  <thead class="gt_col_headings">
    <tr>
      <th class="gt_col_heading gt_columns_bottom_border gt_left" rowspan="1" colspan="1">Parameter</th>
      <th class="gt_col_heading gt_columns_bottom_border gt_center" rowspan="1" colspan="1">Coefficient</th>
      <th class="gt_col_heading gt_columns_bottom_border gt_center" rowspan="1" colspan="1">SE</th>
      <th class="gt_col_heading gt_columns_bottom_border gt_center" rowspan="1" colspan="1">95% CI</th>
      <th class="gt_col_heading gt_columns_bottom_border gt_center" rowspan="1" colspan="1">t(29)</th>
      <th class="gt_col_heading gt_columns_bottom_border gt_center" rowspan="1" colspan="1">p</th>
    </tr>
  </thead>
  <tbody class="gt_table_body">
    <tr><td class="gt_row gt_left">(Intercept)</td>
<td class="gt_row gt_center">39.69</td>
<td class="gt_row gt_center">1.71</td>
<td class="gt_row gt_center">(36.18, 43.19)</td>
<td class="gt_row gt_center">23.14</td>
<td class="gt_row gt_center">&lt; .001</td></tr>
    <tr><td class="gt_row gt_left">wt</td>
<td class="gt_row gt_center">-3.19</td>
<td class="gt_row gt_center">0.76</td>
<td class="gt_row gt_center">(-4.74, -1.64)</td>
<td class="gt_row gt_center">-4.22</td>
<td class="gt_row gt_center">&lt; .001</td></tr>
    <tr><td class="gt_row gt_left">cyl</td>
<td class="gt_row gt_center">-1.51</td>
<td class="gt_row gt_center">0.41</td>
<td class="gt_row gt_center">(-2.36, -0.66)</td>
<td class="gt_row gt_center">-3.64</td>
<td class="gt_row gt_center">0.001 </td></tr>
  </tbody>
  <tfoot class="gt_sourcenotes">
    <tr>
      <td class="gt_sourcenote" colspan="6">hi<br/>Model: mpg ~ wt + cyl (32 Observations)</td>
    </tr>
  </tfoot>
  
</table>
</div>

<sup>Created on 2022-06-02 by the [reprex package](https://reprex.tidyverse.org) (v2.0.1)</sup>

**Comment by DominiqueMakowski** [comment]
> For a start, can it not be red?

if anything, yellow would be better indeed. Red screens warning, and yellow = FYI

**Comment by etiennebacher** [comment]
@mattansb @DominiqueMakowski I have added a commit to put the message in yellow. It won't render in github but you can test it with:
```r
message(insight::format_message("\033[0;33m", "Uncertainty intervals (equal-tailed) and p-values (two-tailed) computed using a Wald t-distribution approximation.", "\033[0m"))
```

**Comment by DominiqueMakowski** [comment]
thanks a lot @etiennebacher 💪  

(et j'en profite pour te (re)souhaiter personellement la bienvenue à easystats on est tous hyper content de t'avoir parmis nous et d'entamer des nouvelles aventures ensemble!)

**Comment by etiennebacher** [comment]
Merci pour le message ;) je me répète mais c'est un ensemble de packages incroyable, j'imagine meme pas tout le travail derrière donc ça me fait plaisir de vous aider au moins un petit peu à le développer et l'améliorer

**Comment by strengejacke** [comment]
> ```r
> parameters::parameters(lm(Sepal.Width ~ Species, data=iris))
> ```

This is no footer, it's a message. And you can omit it with `verbose`:

``` r
parameters::parameters(lm(Sepal.Width ~ Species, data=iris), verbose = FALSE)
#> Parameter            | Coefficient |   SE |         95% CI | t(147) |      p
#> ----------------------------------------------------------------------------
#> (Intercept)          |        3.43 | 0.05 | [ 3.33,  3.52] |  71.36 | < .001
#> Species [versicolor] |       -0.66 | 0.07 | [-0.79, -0.52] |  -9.69 | < .001
#> Species [virginica]  |       -0.45 | 0.07 | [-0.59, -0.32] |  -6.68 | < .001
```

<sup>Created on 2022-06-02 by the [reprex package](https://reprex.tidyverse.org) (v2.0.1)</sup>

I see no need in changing the color.

File: R/format.R [hunk 1]
```diff
@@ -647,6 +647,6 @@ format.parameters_sem <- function(x,
       msg <- paste(msg, "Uncertainty intervals for random effect variances computed using a Wald z-distribution approximation.")
     }
 
-    message(insight::format_message(msg))
+    message(insight::format_message("\033[0;33m", msg, "\033[0m"))
```
Comment (strengejacke) [inline R/format.R:650]: I suggest reverting this change, or at least use `insight::color_text()`, which checks if ANSI-colors are available.

**Review by strengejacke — CHANGES_REQUESTED** [review]

**Comment by strengejacke** [comment]
> For a start, can it not be red?

It's not red, it uses RStudio defaults

File: R/format.R (hunk 1 above)
Comment (etiennebacher) [reply R/format.R:650]: I revert it for now and we can also make another PR later to change the color if needed

**Comment by etiennebacher** [comment]
If you're okay with the fix, I'll just add a news bullet point (and bump version?)

**Comment by strengejacke** [comment]
> can we remove the footer for being printed in the console?

I think @bwiernik made some efforts to make this happen and is happy that we're now fairly transparent about the ci-method, so I guess removing would be a bad idea. ;-)

But, we could use options, as we already do for the summary:

``` r
options("parameters_summary" = TRUE)
parameters::parameters(lm(Sepal.Width ~ Species, data=iris))
#> Parameter            | Coefficient |   SE |         95% CI | t(147) |      p
#> ----------------------------------------------------------------------------
#> (Intercept)          |        3.43 | 0.05 | [ 3.33,  3.52] |  71.36 | < .001
#> Species [versicolor] |       -0.66 | 0.07 | [-0.79, -0.52] |  -9.69 | < .001
#> Species [virginica]  |       -0.45 | 0.07 | [-0.59, -0.32] |  -6.68 | < .001
#> 
#> Model: Sepal.Width ~ Species (150 Observations)
#> Residual standard deviation: 0.340 (df = 147)
#> R2: 0.401; adjusted R2: 0.393
#> 
#> Uncertainty intervals (equal-tailed) and p-values (two-tailed) computed
#>   using a Wald t-distribution approximation.


options("parameters_summary" = FALSE)
parameters::parameters(lm(Sepal.Width ~ Species, data=iris))
#> Parameter            | Coefficient |   SE |         95% CI | t(147) |      p
#> ----------------------------------------------------------------------------
#> (Intercept)          |        3.43 | 0.05 | [ 3.33,  3.52] |  71.36 | < .001
#> Species [versicolor] |       -0.66 | 0.07 | [-0.79, -0.52] |  -9.69 | < .001
#> Species [virginica]  |       -0.45 | 0.07 | [-0.59, -0.32] |  -6.68 | < .001
#> 
#> Uncertainty intervals (equal-tailed) and p-values (two-tailed) computed
#>   using a Wald t-distribution approximation.
```

<sup>Created on 2022-06-02 by the [reprex package](https://reprex.tidyverse.org) (v2.0.1)</sup>

What about having an option for the ci-method message, too?

**Comment by etiennebacher** [comment]
@strengejacke Like this?

``` r
options("parameters_ci_message" = FALSE)
parameters::parameters(lm(Sepal.Width ~ Species, data=iris))
#> Parameter            | Coefficient |   SE |         95% CI | t(147) |      p
#> ----------------------------------------------------------------------------
#> (Intercept)          |        3.43 | 0.05 | [ 3.33,  3.52] |  71.36 | < .001
#> Species [versicolor] |       -0.66 | 0.07 | [-0.79, -0.52] |  -9.69 | < .001
#> Species [virginica]  |       -0.45 | 0.07 | [-0.59, -0.32] |  -6.68 | < .001

options("parameters_ci_message" = TRUE)
parameters::parameters(lm(Sepal.Width ~ Species, data=iris))
#> Parameter            | Coefficient |   SE |         95% CI | t(147) |      p
#> ----------------------------------------------------------------------------
#> (Intercept)          |        3.43 | 0.05 | [ 3.33,  3.52] |  71.36 | < .001
#> Species [versicolor] |       -0.66 | 0.07 | [-0.79, -0.52] |  -9.69 | < .001
#> Species [virginica]  |       -0.45 | 0.07 | [-0.59, -0.32] |  -6.68 | < .001
#> 
#> Uncertainty intervals (equal-tailed) and p-values (two-tailed) computed
#>   using a Wald t-distribution approximation.
```

<sup>Created on 2022-06-02 by the [reprex package](https://reprex.tidyverse.org) (v2.0.1)</sup>

**Comment by strengejacke** [comment]
Yes, I just added a commit that includes this feature. We might ask @DominiqueMakowski which name for the option he prefers, I called it `"parameters_cimethod"`.

**Comment by strengejacke** [comment]
Else, this PR can be merged.

**Comment by etiennebacher** [comment]
Also what is the default for this option? I put TRUE and you FALSE

**Comment by strengejacke** [comment]
> Also what is the default for this option? I put TRUE and you FALSE

Exactly, we shouldn't change the defaults.

**Comment by strengejacke** [comment]
Yeah, my brain is already closing down.

**Pull request merged**